code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -63,6 +63,7 @@
 
 /mob/living/simple_animal/bot/secbot/beepsky/armsky
 	name = "Sergeant-At-Armsky"
+	desc = "It's Sergeant-At-Armsky! A disgruntled assistant to the Warden. He would probably shoot you if he had hands."
```

Comment:
```suggestion
	desc = "It's Sergeant-At-Armsky, a disgruntled assistant to the warden. He would probably shoot you if he had hands."
```
```suggestion
	desc = "It's Sergeant-At-Armsky! He's a disgruntled assistant to the warden that would probably shoot you if he had hands."
```
Slightly clunky wording imo, got options here.

Reply:
I like that second one. Thanks man.